docs/enterprise/concepts.md

```diff
@@ -85,11 +85,15 @@ Routes define the connection pathway and configuration from the internet to your
 
 ### Protected Endpoints
 
-This term refers to the system or service the route provides or restrics access to.
+This term refers to the system or service the route provides or restricts access to.
 
 ## Policies
 
-In the Pomerium Enterprise Console,  FILL ME IN 
+A Policy defines what permissions a set of users or groups has.
+
+Policies can be applied to [Routes](#routes) directly, or enforced within a [Namespace](#namespaces). This associates the set of permissions with a service or set of services, completing the authentication model.
+
+To learn more about how to create Policies in Pomerium Enterprise Console, see [Reference: Policies].
```

Comment:
This section covers Policies as a concept. CC @travisgroth or @desimone if they want to add some more color here.